packages/react/src/primitives/Field/__tests__/FieldDescription.test.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { FieldDescription, QA_FIELD_DESCRIPTION } from '../FieldDescription';
+import { ComponentClassNames } from '../../shared/constants';
+
+describe('FieldDescription component', () => {
+  it('should not render if it has a "labelHidden" prop is `true`', async () => {
```

Comment:
Sorry if I wasn't clear. I think it should render if labelHidden is true, but not if descriptiveText is null. Here's the following scenarios we should have tests for:

1. labelHidden = true, descriptiveText = "Provide a valid USPS address." => FieldDescription with className "amplify-visually-hidden"
1. labelHidden = false, descriptiveText = "Provide a valid USPS address." => FieldDescription in DOM with text shown  "Provide a valid USPS address."
1. labelHidden = true, descriptiveText = null => FieldDescription is not present in DOM
1. labelHidden = false, descriptiveText = null => FieldDescription is not present in DOM

Reply:
Oh, interesting! We want to keep the description even when the label's hidden. Ok...I'll update it. 

Thanks for the detailed clarification!